.github/workflows/ci_vscode.yml

```diff
@@ -65,6 +65,10 @@ jobs:
       - uses: taiki-e/checkout-action@b13d20b7cda4e2f325ef19895128f7ff735c0b3d # v1.3.1
       - uses: ./.github/actions/pnpm
 
+      - name: Build
+        working-directory: editors/vscode
+        run: pnpm run build
```

Comment:
this will build the vscode and the langauge server and costs time in CI.
The goal is to create integrations tests between them.

Reply:
We'll need a follow up PR to combine all the compile + test jobs together to save resources. We have way too many jobs in this org, causing long queue of jobs.